backend/src/core/routes/tests/campaign.routes.test.ts

```diff
@@ -15,6 +15,7 @@ beforeAll(async () => {
 })
 
 afterEach(async () => {
+  await JobQueue.destroy({ where: {} })
   await Campaign.destroy({ where: {}, force: true })
```

Comment:
for my own learning → why is `force: true` needed for `Campaign` and not `JobQueue`?

Reply:
answer: `Campaign` has soft delete, whereas `JobQueue` does not